docs/navigation-configuration.md

```diff
@@ -66,6 +74,23 @@ You can configure the way Luigi tackles routing in your application in the **Rou
 - **nodeAccessibilityResolver** allows you to define a permission checker function that gets executed on every node. If it returns `false`, Luigi removes the node and its children from the navigation structure.
   **nodeAccessibilityResolver** receives all values defined in the node configuration. See [angular sampleconfig.js](../core/examples/luigi-sample-angular/src/assets/extendedConfiguration.js) for the **constraints** example.
 
+## Context Switcher
+
+Allows to switch between a curated list of navigation elements. Adds a drop down switcher next to the profile in the top navigation bar. Typically used for switching between environments. 
```

Comment:
I would replace 'Typically used for switching between environments' with something like 'It could be used e.g. to switch between environments'.

Reply:
I agree with the suggestion, I would just write it a bit differently. So the final sentence would be like:

`The context switcher is a drop-down list available in the top navigation bar. It allows you to switch between a curated list of navigation elements such as Environments.` 

I think we may just write that it's located in the top navigation bar, and it will be enough for the user to find it.